portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,11 @@ function addRandomRecommendation() {
   const recommendationContainer = document.getElementById('recommendation-container');
   recommendationContainer.innerText = recommendation;
 }
+
+function getMessage() {
```

Comment:
Add JSDoc for this function: https://google.github.io/styleguide/jsguide.html#jsdoc